Registry/DirectoryHrefRegistry.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Redeye\StormpathBundle\Registry;
+
+class DirectoryHrefRegistry
```

Comment:
The registry classes are identical, I see little reason to have multiple classes here, a single HrefRegistry class will do fine.